src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -73,12 +73,16 @@ def mds_counters(self, fs_id):
         result = {}
         mds_names = self._get_mds_names(fs_id)
 
+        def __to_second(point):
+            return (point[0] // 1000000000, point[1])
```

Comment:
Why are you doing a floor division `//` instead of rounding?
I think it will be more correct if you do `(round(point[0] / 1000000000.0), point[1])
￼

Reply:
The initial thought is to comply with the original code (https://github.com/ceph/ceph/pull/28882/commits/b758a82dbba0388c7cbff7ed9403985f216e1d65#diff-1c5bf88341c84268fb9146e805ba6fe8L705). `sec()` returns `tv_sec`, so it's a floor operation.

Reply:
ok, sounds good